RomanASP.py

```diff
@@ -75,6 +74,21 @@ def main():
     parser.add_argument('-o', '--output_path', type=str, required=False,
                         help='relative output path')
 
+    parser.add_argument('-c', '--config', type=str, required=False,
+                        help='relative config file path')
+
+    args = parser.parse_args()
+    band = args.band
+    SNID = args.SNID
+    testnum = args.testnum
+    detim = args.detim
+    output_path = args.output_path
+    if args.config is not None:
+        config_path = args.config
+    else:
+        config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                           'config.yaml')
+
```

Comment:
Someday it would be nice to separate out parsing of arguments, loading of configs, and then running the main function.